lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/passportCri/ContractTest.java

```diff
@@ -186,6 +255,28 @@ public RequestResponsePact validRequestReturnsValidAccessToken(PactDslWithProvid
                 .toPact();
     }
 
+    @Pact(provider = "PassportCriProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact invalidAccessTokenRequestReturns400(PactDslWithProvider builder) {
```

Comment:
Please rename - this pact is for an invalid auth code not an invalid access token.